packages/settings-library/src/settings_library/base.py

```diff
@@ -91,7 +91,7 @@ def prepare_field_value(
             and field.default is None
             and prepared_value == _MARKED_AS_UNSET
         ):
-            prepared_value = field.default_factory()
+            prepared_value = field.default_factory()  # type: ignore[call-arg]
```

Comment:
So it's the same here where you call the factory it complains

Reply:
yes it complains because there are two callables